Traceback (most recent call last):
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\nbclient\client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\nbclient\client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\nbclient\client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Tellurium-Winpython-3.7\python-3.7.7.amd64\lib\site-packages\nbclient\client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#%% You can load an SBML model directly from the BioModels Database, given the BioModel ID
# wolf = te.loadSBMLModel("http://www.ebi.ac.uk/biomodels-main/download?mid=BIOMD0000000090")
wolf = te.loadSBMLModel("BIOMD0000000090.xml")
wolf.simulate(0, 100, 1000, ['time', 'oxy']) # note that specific species can be selected for recording concentrations over the timecourse
wolf.plot(figsize = (10, 6), xtitle = 'Time', ytitle = 'Concentration')
 
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
[1;32m<ipython-input-1-ca2d9d9f832d>[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [1;31m#%% You can load an SBML model directly from the BioModels Database, given the BioModel ID[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[0;32m      2[0m [1;31m# wolf = te.loadSBMLModel("http://www.ebi.ac.uk/biomodels-main/download?mid=BIOMD0000000090")[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 3[1;33m [0mwolf[0m [1;33m=[0m [0mte[0m[1;33m.[0m[0mloadSBMLModel[0m[1;33m([0m[1;34m"BIOMD0000000090.xml"[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      4[0m [0mwolf[0m[1;33m.[0m[0msimulate[0m[1;33m([0m[1;36m0[0m[1;33m,[0m [1;36m100[0m[1;33m,[0m [1;36m1000[0m[1;33m,[0m [1;33m[[0m[1;34m'time'[0m[1;33m,[0m [1;34m'oxy'[0m[1;33m][0m[1;33m)[0m [1;31m# note that specific species can be selected for recording concentrations over the timecourse[0m[1;33m[0m[1;33m[0m[0m
[0;32m      5[0m [0mwolf[0m[1;33m.[0m[0mplot[0m[1;33m([0m[0mfigsize[0m [1;33m=[0m [1;33m([0m[1;36m10[0m[1;33m,[0m [1;36m6[0m[1;33m)[0m[1;33m,[0m [0mxtitle[0m [1;33m=[0m [1;34m'Time'[0m[1;33m,[0m [0mytitle[0m [1;33m=[0m [1;34m'Concentration'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;31mNameError[0m: name 'te' is not defined
NameError: name 'te' is not defined

